benchexec/tablegenerator/template.html

```diff
@@ -1423,6 +1430,11 @@
   showScatterPlot();
 };
 
+function toggleFloatThead() {
+    $(".collapsable-header").toggle();
+    $("#dataTable").floatThead('reflow');
```

Comment:
I think an optimal course of action would be to commit the change without the toggle button, and leave toggling for a separate task.